docs/prow/presets.md

```diff
@@ -0,0 +1,27 @@
+| **preset-docker-push-repository**  | It provides the environment variable with the location of the Docker repository.                                                                                 |
+| **preset-build-pr**                | It provides the environment variable with the location of the directory in the Docker repository for storing images. It also sets the **BUILD_TYPE** variable to `pr`. |
+| **preset-build-master**            | It is similar to the **preset-build-pr** Preset, but the **BUILD_TYPE** variable is set to `master`.                                                            |
+| **preset-build-release**           | It is similar to the **preset-build-pr** Preset, but the **BUILD_TYPE** variable is set to `release`.                                                           |
+| **preset-gc-project-env**          | It provides the environment variable with the Google Cloud Platform (GCP) project name.                                                                                              |
+| **preset-gc-compute-envs**         | It provides environment variables with the gcloud compute zone and the gcloud compute region.                                                                   |
+| **preset-sa-vm-kyma-integration**  | It injects credentials for the service account to run integration tests on virtual machines (VMs).                                                              |
+| **preset-sa-gke-kyma-integration** | It injects credentials for the service account to run integration tests on a Google Cloud Engine (GKE) cluster.                                                 |
+| **preset-bot-npm-token**           | It provides an environment variable with a token for publishing npm packages.
+| **preset-sa-kyma-artifacts** | It sets up the service account that has write permissions to the Kyma's artifacts bucket.                     |
+| **preset-docker-push-repository-gke-integration** | It provides the environment variable with the location of the directory in the GCR repository for storing temporary Docker images for the Kyma Installer.                     |
+| **preset-docker-push-repository-test-infra** | It defines the environment variable with the location of the directory in the GCR repository for storing Docker images from the `test-infra` repository.                    |
+| **preset-docker-push-repository-incubator** | It defines the environment variable with the location of the directory in the GCR repository for storing Docker images from all repositories under the `kyma-incubator` organization.                     |
+| **preset-build-console-master** | It defines the environment variable with the location of the directory in the Docker repository for storing Docker images from the `console` repository. It also sets the **BUILD_TYPE** variable to `master`.                  |
+| **preset-bot-github-token** | It sets the environment variable with the GitHub token.                     |
```

Comment:
To the end, we can add something like "for the bot account" to make it more obvious.